fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/Select.java

```diff
@@ -48,6 +50,12 @@
     // offset/limit for pagination
     private PaginationClause paginationClause;
 
+    // Another Select to UNION with this select. Optional
+    private Select union;
+
+    // If true, the specified UNION is a UNION ALL
+    private boolean unionAll = false;
```

Comment:
is UNION ALL just an optimization or we need duplicates after the union somewhere?

Reply:
UNION ALL will not remove duplicates, while UNION will. This flag indicates if the Select statement in the 'union' field should be a UNION ALL or a UNION. Just gives the caller the option to choose which they want to use.